### PR TITLE
For 'extract-xmsexamples', allowing windows-style file-paths for '<sp…

### PR DESCRIPTION
**Pull request by manaas-microsoft**
…ec-path>' and '<recordings>'.

### CODE REVIEW

**Comment by manaas-microsoft** [comment]
@veronicagg Hi, Was trying to generate x-ms-examples using this script. Since, I was using it on windows, hit issues related to path delimiters being "\\" instead of "/". Have made that change, and added here. Please do let me know if this looks fine.

File: lib/xMsExampleExtractor.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ class xMsExampleExtractor {
                   let infoFromOperation = paths[path][requestMethodFromRecording.toLowerCase()];
                   if (typeof infoFromOperation != 'undefined') {
                     //need to consider each method in operation
-                    let fileName = recordingFileName.split('/');
+                    let fileName = recordingFileName.split(/[/\\]/);
```
Comment (vishrutshah) [inline lib/xMsExampleExtractor.js]: Should you use `pathlib.sep` instead?

File: lib/xMsExampleExtractor.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ class xMsExampleExtractor {
 
     let outputExamples = self.options.output + "/examples/";
     let relativeExamplesPath = "../examples/";
-    let specName = self.specPath.split("/");
+    let specName = self.specPath.split(/[/\\]/);
```
Comment (vishrutshah) [inline lib/xMsExampleExtractor.js]: same question here

File: lib/xMsExampleExtractor.js (hunk 1 above)
Comment (manaas-microsoft) [inline lib/xMsExampleExtractor.js]: Sure. Will check that. But I wanted to allow both kind of inputs "windows-style" and "Linux-style" to be parsed successfully. Will check if that happens using above option.

**Comment by amarzavery** [comment]
closing this PR due to age. Please feel free to open if required.

**Pull request closed without merging**